sample/SipInterconnect/README.md

```diff
@@ -0,0 +1,238 @@
+
+
+
+## Prerequisites
+
+To be prepared to develop your OpenTok SIP Interconnect app:
+
+1. Review the [OpenTok.js](https://tokbox.com/developer/sdks/js/) requirements.
+2. Your app will need **Token** and **API Key**, which you can get at the [OpenTok Developer Dashboard](https://dashboard.tokbox.com/). Set the API Key and API Secret in [config.js](./config.js).
+
+
+To install the OpenTok SIP Interconnect Sample App, run the following commands:
+
+```
+npm install
+node bin/www
+```
+
+
+_**IMPORTANT:** In order to deploy an OpenTok SPI Interconnect app, your web domain must use HTTPS._
+
+
+## Quick start
+
+The web page that loads the sample app for JavaScript must be served over HTTP/HTTPS. Browser security limitations prevent you from publishing video using a `file://` path, as discussed in the OpenTok.js [Release Notes](https://www.tokbox.com/developer/sdks/js/release-notes.html#knownIssues). To support clients running [Chrome 47 or later](https://groups.google.com/forum/#!topic/discuss-webrtc/sq5CVmY69sc), HTTPS is required. A web server such as [MAMP](https://www.mamp.info/) or [XAMPP](https://www.apachefriends.org/index.html) will work, or you can use a cloud service such as [Heroku](https://www.heroku.com/) to host the application.
+
+
+
+## Exploring the code
+
+This section describes how the sample app code design uses recommended best practices to deploy the SIP Interconnect features. 
+
+For detail about the APIs used to develop this sample, see the [OpenTok.js Reference](https://tokbox.com/developer/sdks/js/reference/).
+
+  - [Web page design](#web-page-design)
+  - [SIP Call](#sip-call)
+
+
+_**NOTE:** The sample app contains logic used for logging. This is used to submit anonymous usage data for internal TokBox purposes only. We request that you do not modify or remove any logging code in your use of this sample application._
+
+### Web page design
+
+While TokBox hosts [OpenTok.js](https://tokbox.com/developer/sdks/js/), you must host the sample app yourself. This allows you to customize the app as desired. For details about the one-to-one communication audio-video aspects of the design, see the [OpenTok One-to-One Communication Sample App](https://github.com/opentok/one-to-one-sample-apps/tree/master/one-to-one-sample-app/js) and [OpenTok Common Accelerator Session Pack](https://github.com/opentok/acc-pack-common/).
+
+* **[index.ejs](./views/index.ejs)**: This defines the container defining the publisher and layout for the sample app, including the buttons to start and end the SIP call. This file contains the business logic that initiates the OpenTok session and handles the SIP call click events.
+
+* **[config.js](./config.js)**: Configures the participant’s **Session ID**, **Token**, and **API Key** used to create the OpenTok session in [index.js](./routes/index.js), and specifies the SIP headers and credentials required to start the SIP call.
```

Comment:
this feels extraneous. they are in the education path, do we really want to divert them to other paths before they've learned anything about this implementation?